source/common/dmtbinfo2.c

```diff
@@ -678,6 +678,77 @@ ACPI_DMTABLE_INFO           AcpiDmTableInfoLpit0[] =
     ACPI_DMT_TERMINATOR
 };
 
+/* 17: core interrupt controller */
+ACPI_DMTABLE_INFO           AcpiDmTableInfoMadt18[] =
+{
+    {ACPI_DMT_UINT8,   ACPI_MADT18_OFFSET (Version),           "Version", 0},
```

Comment:
This formatting is all over the place

Reply:
Each apic structure for LoongArch has a 'Version' element so that each one can be updated seperately in future.

Thanks.

Reply:
Code formatting, ie whitespace, nothing lines up

Reply:
Ok, got it,  I'll format  these code, thank you very much.